### DISCUSSION TITLE
Realtime API filter

### DISCUSSION THREAD

**Discussion by yamadaOliva** [pocketbase/pocketbase#5377] [Q&A]
i have product collection, i wanna use realtime api for record in this collection that records have type = "test". What should i do ?

**Comment by ganigeorgiev** [comment]
It is not clear what you are asking, nor what you are trying to do.

See the reply from one of your other questions in https://github.com/pocketbase/pocketbase/discussions/5218.

If that doesn't help, please don't treat the discussions as llm chat and try to elaborate a little more on your use case and what is desired behavior, otherwise it is just lazy and low effort and I doubt that someone will be able to assist you.

**Reply by yamadaOliva** [reply]
I'm sorry I seem to have forgotten I asked this before

**Reply by yamadaOliva** [reply]
i used API realtime not like database sir

**Reply by ganigeorgiev** [reply]
Sorry, I don't think I'll be able to help you. This doesn't really explain anything to me.
_I'll close the other discussion as duplicated for this one._

**Reply by yamadaOliva** [reply]
![image](https://github.com/user-attachments/assets/66a9215d-441b-4208-8574-7e814f5a624a)
this's my body API
